src/components/registration/RegistrationForm.tsx

```diff
@@ -123,40 +275,41 @@ const RegistrationForm = ({ onSubmit }: RegistrationFormProps) => {
       <Typography component="h3" variant="h5">
         Children
       </Typography>
-      <FormFieldGroup
-        testId={TestId.ChildrenDefaultFields}
-        fields={getDefaultStudentFields(StudentRole.CHILD)}
-        onChange={(data) => setChildData(Object.assign(childData, data))}
-      />
-      <FormFieldGroup
-        testId={TestId.ChildrenDynamicFields}
-        fields={childDynamicFields}
-        onChange={(data) =>
-          setChildData(
-            Object.assign(childData, { ...childData, information: data })
-          )
-        }
-      />
+      {childData.map((child, i) => (
+        <ChildrenRegistrationForm
+          id={i}
+          key={child.id}
+          dynamicFields={childDynamicFields}
+          defaultFields={getDefaultStudentFields(StudentRole.CHILD)}
+          childData={childData}
+          updateChild={updateChild}
+          handleDeleteChild={handleDeleteChild}
+        />
+      ))}
+      <Button onClick={handleAddChild}>Add Child</Button>
```

Comment:
(being a lil nitpicky so this plays nicely with the refactor comin up) can we put all of this logic within the student registration form component?